.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: go
-sudo: false
```

Comment:
`sudo` is a deprecated tag that has no effect ([details](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration))